pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -88,16 +93,23 @@ var (
 	`)
 
 	volumeUsageCheckerScript = `
-echo "volume percentage checker started....."
+echo "[disk usage checker] Started"
+target_dir="%s"
+usage_percentage_limit="%d"
 while true; do
-disk_usage=$(du -s "%s" | awk '{print $1}')
-disk_space=$(df -P "%s" | awk 'NR==2 {print $2}')
+disk_usage=$(du -s "$target_dir" | awk '{print $1}')
```

Comment:
Although this change makes sense and we can discuss this (with the possible effects on the current usages with respect to backwards compatibility), I think this is not the goal of this PR? or I'm missing something?